# mrMLM
All the seven multi-locus GWAS methods are available to identify QTNs, including a new BLUPmrMLM method that has fast computational speed (similar to EMMAX and GEMMA), high power and accuracy, and low false positive rate.

Conduct multi-locus genome-wide association study under the framework of multi-locus random-SNP-effect mixed linear model (mrMLM). First, each marker on the genome is scanned for the first six methods, while vectorized Wald tests, along with optimal variable selection, are conducted for identifying potentially associated markers for BLUPmrMLM. Bonferroni correction is replaced by a less stringent selection criterion for significant test. Then, all the markers that are potentially associated with the trait are included in a multi-locus genetic model, their effects are estimated by empirical Bayes, and all the nonzero effects were further identified by likelihood ratio test for significant QTL. The program may run on a desktop or laptop computers. If marker genotypes in association mapping population are almost homozygous, these methods in this software are very effective. If there are many heterozygous marker genotypes, the IIIVmrMLM software is recommended. Wen YJ, Zhang H, Ni YL, Huang B, Zhang J, Feng JY, Wang SB, Dunwell JM, Zhang YM, Wu R (2018, <doi:10.1093/bib/bbw145>), and Li M, Zhang YW, Zhang ZC, Xiang Y, Liu MH, Zhou YH, Zuo JF, Zhang HQ, Chen Y, Zhang YM (2022, <doi:10.1016/j.molp.2022.02.012>).

The current version is v5.0.2 (2023-04-06) and the previous versions are available at https://cran.r-project.org/web/packages/mrMLM/index.html.
